indico/MaKaC/PDFinterface/conference.py

```diff
@@ -94,15 +95,12 @@ def enumerate_affil(person_links):
 
 class ProgrammeToPDF(PDFBase):
 
-    def __init__(self, conf, doc=None, story=None, tz=None):
-        self._conf = conf
-        self._event = conf.as_event
-        if not tz:
-            self._tz = self._conf.getTimezone()
-        else:
-            self._tz = tz
-        PDFBase.__init__(self, doc, story)
-        self._title = _("Conference Scientific Programme")
+    def __init__(self, event, tz=None):
+        self._event = event
+        self._conf = event.as_legacy
+        self._tz = self._event.tzinfo
+        self._title = get_menu_entry_by_name('program', self._conf).localized_title
+        PDFBase.__init__(self, title='program.pdf')
```

Comment:
I don't know where `PDFLaTeXBase` gets the PDF title from, but for `contributions/contributions.pdf` the title metadata is `contributions.pdf` (as it gets displayed in the browser tab). In `PDFBase` we were simply not passing `title` as I just added the argument yesterday, so the tab name was always `(anonymous)`.